components/engagement/EngComments.vue

```diff
@@ -73,43 +51,58 @@ export default {
   }),
 
   methods: {
-    goBack() {
-      this.$router.back()
-    },
-
-    submitForm(commentInfo) {
-      this.$store.dispatch('engagements/addComment', commentInfo)
+    submitComment() {
+      if (this.newComment !== '') {
```

Comment:
Just use this.newComment. Check out [this](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) JS concept